packages/h5wasm/src/models.ts

```diff
@@ -4,7 +4,7 @@ import type {
   CompoundTypeMetadata,
 } from 'h5wasm/src/hdf5_util_helpers';
 
-export type H5WasmEntity = NonNullable<ReturnType<H5WasmGroup['get']>>;
+export type H5WasmEntity = ReturnType<H5WasmGroup['get']>;
```

Comment:
Still needed?

Reply:
Well, yes, it is handy and clearer to put `H5WasmEntity` rather `NonNullable<ReturnType<H5WasmGroup['get']>>` (when typing argument from guards for example)

Reply:
Sorry, I meant removing the non-nullable part, but all good 👍

Reply:
Perhaps the diff is not clear but it was indeed removed